qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -102,7 +100,7 @@ public void getPossibleKmerStringTest()  {
 	}
 	
 	@Test
-	public void bothReversedTest() throws IOException {		
+	public void bothReversedTest() throws IllegalArgumentException, IOException {		
```

Comment:
IllegalArgumentException is an unchecked exception and so you don't need it in the throws clause